string location_name
bool wait_for_success
---
bool success